jest/jest-tests.ts

```diff
@@ -199,6 +199,16 @@ describe('Assymetric matchers', function () {
         }));
 
         expect.assertions(4);
+
+        interface Test {
+            a: number;
+            b: string;
+        }
+
+        const test: Test = {
```

Comment:
Could you flesh out the rest of this use case here in the test? It's a little abstract as-is. I'm guessing it's going to end up as `expect(x).toBeCalledWith(test)`?